acme/acme/crypto_util.py

```diff
@@ -161,31 +160,23 @@ def _pyopenssl_cert_or_req_san(cert_or_req):
     :rtype: `list` of `unicode`
 
     """
-    # constants based on implementation of
-    # OpenSSL.crypto.X509Error._subjectAltNameString
-    parts_separator = ", "
+    # constants based on PyOpenSSL certificate/CSR text dump
```

Comment:
There's [this](https://github.com/letsencrypt/letsencrypt/pull/1943/files#diff-f437033746ea336ab13f6ba632e01807R172). Inadequate?